lib/src/flutter_deck_app.dart

```diff
@@ -143,6 +143,12 @@ class _FlutterDeckAppState extends State<FlutterDeckApp> {
     _router = _flutterDeckRouter.build();
   }
 
+  @override
+  void reassemble() {
+    super.reassemble();
+    _router = _flutterDeckRouter.build();
```

Comment:
@mkobuolys look like just call build `_flutterDeckRouter.build()` works, so i didn't call `_buildRouter()` to rebuild the entiry router, do you think it still need to call `_buildRouter()`?